### PR TITLE
Rebuild /openstack page 

### PR DESCRIPTION
**Pull request by bethcollins92**
## Done

- Rebuild page at `/openstack`

## QA

- Check out this feature branch
- Run the site using the command `./run serve` or `dotrun`
- View the site locally in your web browser at: http://0.0.0.0:8001/openstack
- Run through the following [QA steps](https://canonical-web-and-design.github.io/practices/workflow/qa-steps.html)
- Check page against [copy doc](https://docs.google.com/document/d/1eqHK0hc9v7Mn-XV36QRyNkJenTXgBQR_-RULKm5rZ5o/edit#). I've suggested a change to the hero text to keep it more inline with the rest of ubuntu.com. 
- The graph mentioned with the slider will be done as a separate [issue](https://github.com/canonical-web-and-design/ubuntu.com/issues/9524) as this will take some time. 

## Issue / Card

Fixes [#3821](https://github.com/canonical-web-and-design/web-squad/issues/3821)

### CODE REVIEW

**Comment by anasereijo** [comment]
Hi @bethcollins92 I have some amendments:

- [ ] check copy doc for some heading updates throughout the page

- [ ] 'We bring infrastructure costs down' section: can you restyle it to look like the following image? (check [this](https://ubuntu.com/0_docs/content_strips))
Screenshot 2021-04-12 at 10 53 56

- [ ] move quote below that section

- [ ] 'Fixed-price delivery' and 'Fully-managed OpenStack' pricing cards: please match with the following (taken from [this](https://ubuntu.com/openstack) page)
Screenshot 2021-04-12 at 11 01 24

- [ ] 'Security maintenance and support' sub heading rows: remove the indents, it looks misaligned now and use the [coloured rows pattern](https://vanillaframework.io/docs/examples/patterns/tables/table-colored-rows) using #666 as background

- [ ] remove the dash - 
Screenshot 2021-04-12 at 10 51 44

**Comment by bethcollins92** [comment]
Thanks @anasereijo - ready for re-review 👍🏼

**Comment by anasereijo** [comment]
@bethcollins92 this is looking great, thanks!

You only forgot to update the price card for 'Fully-managed OpenStack' section.

**Comment by bethcollins92** [comment]
Thanks, good spot @anasereijo - it's ready (again)

**Comment by anasereijo** [comment]
perfect, thanks

File: templates/openstack/index.html [hunk 1]
```diff
@@ -24,346 +24,691 @@ <h1>OpenStack, delivered</h1>
...
+<section class="p-strip--light">
   <div class="row">
-    <div class="col-7">
-      <p>Canonical operates many private OpenStack clouds on behalf of customers, enabling them to get the benefits of OpenStack while focusing their team on business workloads rather than infrastructure.</p>
-      <p>This is the fast way to validate the economic case for OpenStack; you don't need new staff or skills, just allocate the hardware and engage with Canonical to manage your OpenStack. Once you have proven the economics of private cloud, you can invest in the skills and time to operate it yourself.</p>
-      <p>Fully managed OpenStack is a 'build, operate, transfer, support' service that allows you to take control of your own cloud at any time. All Openstack and operational components deployed are open source.</p>
+    <div class="col-6">
+      <h2>Engineered for best price&ndash;performance</h2>
+      <h3 class="p-heading--4">Designed to be economical in every way</h3>
+      <p>Smart operations, optimal architecture, better pricing. All of this to ensure TCO reduction of your cloud, while expanding your budget for innovation.</p>
```
Comment (anthonydillon) [inline templates/openstack/index.html:238]: This is two sentences in the copy doc.

File: templates/openstack/index.html [hunk 2]
```diff
@@ -24,346 +24,691 @@ <h1>OpenStack, delivered</h1>
...
+        <tbody>
+          <tr>
+            <td>
+              <aside class="p-accordion">
+                <ul class="p-accordion__list">
+                  <li class="p-accordion__group">
+                    <div role="heading" aria-level="3" class="p-accordion__heading">
+                      <button type="button" class="p-accordion__tab" id="tab1" aria-controls="tab1-section" aria-expanded="false">c1.small</button>
+                    </div>
+                    <section class="p-accordion__panel" id="tab1-section" aria-hidden="true" aria-labelledby="tab1">
```
Comment (anthonydillon) [inline templates/openstack/index.html:305]: I think we need to override the padding in this section. Could you add `style:"padding-left: 1rem;"`

Comment (anthonydillon) [reply templates/openstack/index.html:305]: On all the panels

**Review by anthonydillon — APPROVED** [review]
LGTM a few small suggestions in the code but good to go once addressed

**Pull request merged**